arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -821,6 +821,7 @@
 		F140E1EE23F36707008AC44E /* DisplayAnnotationViewController.swift in CopyFiles */ = {isa = PBXBuildFile; fileRef = F140E1EA23F35E92008AC44E /* DisplayAnnotationViewController.swift */; };
 		F14A8F012474AD9D00242D96 /* ConfigureSubnetworkTrace.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = F14A8F002474AD9D00242D96 /* ConfigureSubnetworkTrace.storyboard */; };
 		F14F2CA323FC768200693196 /* DisplaySubtypeSettingsViewController.swift in CopyFiles */ = {isa = PBXBuildFile; fileRef = F1987D0223F7677E00DF41CC /* DisplaySubtypeSettingsViewController.swift */; };
+		F1510FD3254A18E200C29150 /* dodge_city.vtpk in Resources */ = {isa = PBXBuildFile; fileRef = F1510FD2254A18E200C29150 /* dodge_city.vtpk */; settings = {ASSET_TAGS = (DodgeCity, ); }; };
```

Comment:
Hmm it seems during the update the source code and README are dropped. Could you add them back?

Reply:
I don't see any files that we deleted or missing. Which files are you referring to?

Reply:
The README and source code. When tapping on the upper right info button, the README doesn't load. Correct me if I'm wrong and I can take another look.

Reply:
![Simulator Screen Shot - iPhone 12 Pro Max - 2020-11-05 at 16 41 33](https://user-images.githubusercontent.com/52215269/98312125-df283e00-1f85-11eb-86bd-312f5bfee7cd.png)
![Simulator Screen Shot - iPhone 12 Pro Max - 2020-11-05 at 16 41 35](https://user-images.githubusercontent.com/52215269/98312127-e0f20180-1f85-11eb-9eb9-ff651bba3309.png)
Do you mean the info button in the Simulator? They loaded okay for me

Reply:
I see. I'll take another look in next review! 😳